Zend/tests/ErrorException_getSeverity.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+final public int ErrorException::getSeverity ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - #phparty7 - @phpsp - novatec/2015 - sao paulo - br
+--FILE--
+<?php
+function exception_error_handler($severity) {
+ 
+    $message = "error exception, severity =>";
+    $code = 500;
+    $severity = 75;
+    try{
+        if (!(error_reporting() & $severity)) {
+            return;
+        }
+        throw new ErrorException($message, $code, $severity);
+    } catch(ErrorException $err) {
+        var_dump(
```

Comment:
Instead of this long var_dump(), you can also test for \Exception's magic __toString() method by calling:

echo $err;